massa-node/base_config/config.toml

```diff
@@ -217,8 +217,8 @@
     broadcast_slot_execution_output_channel_capacity = 5000
     # slot execution traces channel capacity
     broadcast_slot_execution_traces_channel_capacity = 5000
-    # Max execution traces (group by block id) to keep in cache
-    execution_traces_limit = 5000
+    # Max slots execution traces to keep in cache
+    execution_traces_limit = 320
```

Comment:
There is still a value of 5000 in the default config for execution config for tests : https://github.com/massalabs/massa/blob/lower_sizes_execution_trace/massa-execution-exports/src/test_exports/config.rs#L74

Reply:
done